tests/gcp/deployment/google_folder/generate_folder_import_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+FOLDER_FILES_PATH=$(pwd)
+TF_VAR_sub_folder_parent=${TF_VAR_sub_folder_parent:-staging}
```

Comment:
Uses a variable for the parent folder for the `sub_folder` so that the script can obtain the correct parent_id, and therefore use `gcloud` to get the folder id of the sub_folder (`gcloud resource-manager folders list` requires a parent folder or organization as an argument)